flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSink.java

```diff
@@ -378,16 +371,14 @@ public void testOverrideWriteConfigWithUnknownFileFormat() {
             .writeParallelism(parallelism)
             .setAll(newProps);
 
-    AssertHelpers.assertThrows(
-        "Should fail with invalid file format.",
-        IllegalArgumentException.class,
-        "Invalid file format: UNRECOGNIZED",
-        () -> {
-          builder.append();
-
-          // Execute the program.
-          env.execute("Test Iceberg DataStream.");
-          return null;
-        });
+    Assertions.assertThatThrownBy(
+            () -> {
+              builder.append();
+
+              // Execute the program.
+              env.execute("Test Iceberg DataStream.");
```

Comment:
I think we should remove the call to `env.execute("Test Iceberg DataStream.");` because it implies that the error happens on that call, whereas in fact it happens when calling `builder.append()`

```
Assertions.assertThatThrownBy(builder::append)
        .isInstanceOf(IllegalArgumentException.class)
        .hasMessage("Invalid file format: UNRECOGNIZED");
```

Reply:
fixed